docs/components/email/index.html

```diff
@@ -0,0 +1,143 @@
+              Optional
+            </label>
+          </hx-checkbox-control>
+
+        </fieldset>
+
+        <fieldset>
+          <legend class="beta-hxFieldName">Control Options</legend>
+          <hx-checkbox-control>
+            <input id="chkDisabled" type="checkbox" v-model="isDisabled" />
+            <label for="chkDisabled">
+              <hx-checkbox></hx-checkbox>
+              Disabled
+            </label>
+          </hx-checkbox-control>
+
+          <hx-checkbox-control>
+            <input id="chkRequired" type="checkbox" v-model="isRequired" />
+            <label for="chkRequired">
+              <hx-checkbox></hx-checkbox>
+              Required
+            </label>
+            <p>
+              (Invalid styling applied <i>after</i> user interaction.)
+            </p>
+          </hx-checkbox-control>
+        </fieldset>
+        <fieldset>
+          <legend class="beta-hxFieldName">Optional Text</legend>
+          <hx-checkbox-control>
+            <input id="chkHasHelpText" type="checkbox" v-model="hasHelpText" />
+            <label for="chkHasHelpText">
+              <hx-checkbox></hx-checkbox>
+              HelpText
+            </label>
+          </hx-checkbox-control>
+          <hx-checkbox-control>
+              <input id="chkHasErrorText" type="checkbox" v-model="hasErrorText" />
+              <label for="chkHasErrorText">
+                <hx-checkbox></hx-checkbox>
+                ErrorText
+              </label>
+            </hx-checkbox-control>
+            <hx-text-control class="align" > 
+              <label> HelpText</label>
+            <input  type="text" v-model="helpText" :disabled="!helpTextToDisplay" >
+           </hx-text-control>
+           <hx-text-control class="align" >
+              <label> ErrorText</label>
+            <input  type="text" v-model="errorText" :disabled="!errorTextToDisplay"> 
+           </hx-text-control>
+          </fieldset>
+      </form>
+    </header>
+
+    <div>
+      <hx-email-control>
+          <!-- <label >
+              {% raw %}{{helptext}}{% endraw %}
+            </label> -->
+        <input id="emailInputDemo" type="email" placeholder="Please Enter Email" :disabled="isDisabled" :required="isRequired" />
+        <label for="emailInputDemo" :class="{ hxOptional: hasOptional, hxRequired: hasAsterisk }">
+          {% raw %}{{label}}{% endraw %}
+        </label>
+        <span>
+            Please enter a valid email ID
+           </span>
+       
+      </hx-email-control>
+      <hx-checkbox-control>
+          <p :class="{ hxHelpText: hasHelpText}"  v-if="helpTextToDisplay">
+             {% raw %}{{helpText}} {% endraw %}
+         </p>
+      </hx-checkbox-control>
+      <hx-checkbox-control >
+         <p :class="{ hxErrorText: hasErrorText}" v-if="errorTextToDisplay">
+            <hx-icon type="exclamation-triangle"></hx-icon>
+            {% raw %}{{errorText}} {% endraw %}
+        </p>
+        </hx-checkbox-control>
```

Comment:
1. Please remove this `<hx-checkbox-control>`.
1. Move the `<p>` into `<hx-password-control>`.
1. Update the logic to toggle the `hidden` attribute on `<p>`.
1. Keep the `hxErrorText` class, and toggle `hidden`.

Reply:
Changes have been done, please review.

Reply:
I'm sorry I had a typo in the above.  Please move the `<p>` into the `<hx-email-control>` demo component.